packages/cli/lib/base-generator.js

```diff
@@ -257,6 +257,18 @@ module.exports = class BaseGenerator extends Generator {
     this.exitGeneration = reason;
   }
 
+  /**
+   * Run `npm install` in the project
+   */
+  install() {
```

Comment:
❗️ This is moved as is from `project-generator` to the `base-generator`. ProjectGenerator extends BaseGenerator. Done so this can be leveraged by Artifact Generators such as the `ExampleGenerator`.